src/content/developers/docs/standards/tokens/erc-777/index.md

```diff
@@ -5,6 +5,10 @@ lang: en
 sidebar: true
 ---
 
+## Deprecation Notice {#deprecation-notice}
+
+**ERC-777 is deprecated. It is recommended to use [ERC-20](/developers/docs/standards/tokens/erc-20/) instead.** This page remains as a historical archive.
```

Comment:
I think this is a strong statement and makes ethereum.org very opinionated. Who decides "deprecation"?

A better way would be pointing to some "expert discussion" stating "ERC-777 is not recommended by some/many due to potential security issues".

Reply:
I'll change the wording.